java/java.lsp.server/vscode/.vscode/launch.json

```diff
@@ -31,7 +31,10 @@
 			"outFiles": [
 				"${workspaceFolder}/out/test/**/*.js"
 			],
-			"preLaunchTask": "${defaultBuildTask}"
+			"preLaunchTask": "${defaultBuildTask}",
+			"env": { 
+				"netbeans.extra.options" : "-J-Dproject.limitScanRoot=${workspaceFolder}/out"
```

Comment:
An API to document as `<api category="systemproperty" />` somewhere.

Reply:
OK, documented in project api's `arch.xml`